app/api/migrations/migrations/122-update_translations/removed_keys.csv

```diff
@@ -0,0 +1,2 @@
+key
+Value cannot be transformed to date
```

Comment:
I think it's not needed to have files at all, it could be perfectly just an array in the index file since there is just one entry for deleted keys and added keys. The section for keysToUpdate can be deleted totally also.